lib/scope.js

```diff
@@ -22,6 +22,17 @@ let escape_regex = function( to_escape ) {
   return to_escape.replace(/[-\/\\^$*+?.()|[\]{}]/g, '\\$&');
 };
 
+let display_var_value = function(row) {
```

Comment:
Maybe something with "printable" or "public" instead of "display"? I know we use "display" a lot in AL core, but I generally find it a bit confusing.

Reply:
printable sounds good. I can change it to that.